### PR TITLE
Early may updates

### PR DESCRIPTION
**Pull request by asgrim**
- Nexmo to sponsors
 - RootScope to sponsors
 - Hands on space info for Rowan

### CODE REVIEW

File: schedule.html [hunk 1]
```diff
@@ -246,7 +265,7 @@ <h2 id="saturday">Saturday 10th June</h2>
         </tr>
         <tr>
             <td>14:50</td>
-            <td colspan="2">
+            <td>
```
Comment (lboynton) [inline schedule.html:269]: Should this not be a colspan 2?

File: schedule.html [hunk 2]
```diff
@@ -185,22 +188,19 @@ <h2 id="friday">Friday 9th June</h2>
...
         </thead>
         <tbody>
         <tr>
-            <td>--</td>
-            <td>Sleep! No early morning hangover slot here :)</td>
-        </tr>
-        <tr>
             <td>09:00</td>
-            <td>
+            <td colspan="3">
```
Comment (lboynton) [inline schedule.html]: Shouldn't this be 2?

**Review by lboynton — COMMENTED** [review]
Probably all ok, difficult to see on mobile

File: schedule.html (hunk 2 above)
Comment (asgrim) [reply schedule.html]: That's what I thought, but it didn't stretch across for some reason *shrug*

File: schedule.html (hunk 1 above)
Comment (asgrim) [reply schedule.html:269]: It over-stretched on colspan 2 fo rosmerason

File: schedule.html (hunk 2 above)
Comment (lboynton) [reply schedule.html]: Possibly due to a bad rowspan value somewhere?

File: schedule.html [hunk 3]
```diff
@@ -209,6 +209,25 @@ <h2 id="saturday">Saturday 10th June</h2>
...
+                    experts.
+                </p>
+                <p>
+                    There'll be tables for you to sit down and get stuck in, and we'll post a timetable of who you can
+                    expect to work with during each hour of the day, alongside the great talks in the main track.
+                </p>
+                <p>
+                    Slots will be available to anyone who wants to take part - perhaps you'd like to give a face-to-face
+                    tutorial on your favourite framework, or show people how easy it is to contribute to your team's
+                    open source project? You don't have to pick a time now, but if you're interested, please contact us
```
Comment (lboynton) [inline schedule.html]: Link to contact form might be a good idea

File: schedule.html (hunk 1 above)
Comment (lboynton) [reply schedule.html:269]: This is probably due to the rowspan above

File: schedule.html (hunk 2 above)
Comment (lboynton) [reply schedule.html]: It might look inconsistent across browsers if left like this

File: schedule.html (hunk 3 above)
Comment (asgrim) [reply schedule.html]: Added

**Review by lboynton — APPROVED** [review]
Well it's probably ok 😇

File: schedule.html (hunk 1 above)
Comment (asgrim) [reply schedule.html:269]: No it shouldn't be colspan 2 (the track 2 column spans the whole day including lunch & breaks)

File: schedule.html (hunk 2 above)
Comment (asgrim) [reply schedule.html]: Yeah it works now with colspan 2

**Pull request merged**